internal/transactions/transactions.go

```diff
@@ -52,16 +54,27 @@ type TransactionResult struct {
 
 // JSON convert result to JSON
 func (r *TransactionResult) JSON() interface{} {
-	result := make(map[string]string)
+	result := make(map[string]interface{})
 	result["id"] = r.tx.ID().String()
 	result["payload"] = fmt.Sprintf("%x", r.tx.Encode())
 	result["authorizers"] = fmt.Sprintf("%s", r.tx.Authorizers)
 	result["payer"] = r.tx.Payer.String()
 
 	if r.result != nil {
-		result["events"] = fmt.Sprintf("%s", r.result.Events)
 		result["status"] = r.result.Status.String()
 
+		txEvents := make([]interface{}, 0)
```

Comment:
Maybe allocate the expected capacity

```suggestion
		txEvents := make([]interface{}, 0, len(r.result.Events))
```

Reply:
thank you for suggestion, fixed